tests/test_cmdline.py

```diff
@@ -917,7 +918,8 @@ def test_cmdline_on_ignored_undeclared_option(
                 # code = ['.']
                 deps = ['foobar']
                 # pyenv = None
-                # custom_mapping = None
+                # custom_mapping_file = None
+                # [tool.fawltydeps.custom_mapping]
```

Comment:
Hmm. This should be at the bottom og the generated text, as otherwise, the options below will be considered part of this new section 😱